UIRes/pluginmetadata.json

```diff
@@ -0,0 +1,77 @@
+[
+  {
+	"Name": "AccurateCountDown",
+    "DevSupportState": "Obsolete",
+    "DevSupportStateReason": "Replaced by EngageTimer"
+  },
+  {
+    "Name": "ACTEndcounter",
+    "DevSupportState": "Adoptable"
+  },
+  {
+    "Name": "BrowserHost.Plugin",
+    "DevSupportState": "Obsolete",
+    "DevSupportStateReason": "Replaced by Browsingway"
+  },
+  {
+    "Name": "CharSyncPlugin",
```

Comment:
This was actually a replacement/alternative for Character Data Sync iirc. It's outdated, but I'm not sure this reason matches up officially.